(WIP) Mtopo27/size analysis docs (#15256)
getsentry/sentry-docs: includes/size-analysis/section-cli.mdx

> Set the GitHub-provided metadata automatically where possible. When running locally, supply the fields explicitly.

Reviewer: ```suggestion
On GitHub Actions metadata is applied automatically. When running in other environments, the fields must be supplied explicitly. 
```
